guidance/models/transformers/_transformers.py

```diff
@@ -63,37 +91,45 @@ def __init__(self, model, tokenizer, chat_template=None, ignore_bos_token=False)
 
             # run a quick spot check to verify we can rebuild complex multi-token unicode symbols
             s = "’•¶∂ƒ˙∆£Ħ爨ൠᅘ∰፨"
-            t = tokenizer
             reconstructed = b""
             try:
-                for i in t(s)["input_ids"]:
-                    reconstructed += bytes(
-                        [byte_decoder[c] for c in t.convert_ids_to_tokens(i)]
-                    )
+
+                for i in transformers_tokenizer(s)["input_ids"]:
+                    nxt_bytes = []
+                    for c in transformers_tokenizer.convert_ids_to_tokens(i):
+                        nxt_bytes.append(byte_decoder[c])
+                    reconstructed += bytes(nxt_bytes)
                 # Check if the tokenizer has a bos_token attribute, and if it does, check
                 # if it's at the start of the reconstructed bytes
                 # Some tokenizers add this automatically as part of the call function, so
                 # we need to remove it to compare
-                if hasattr(t, "bos_token") and reconstructed.startswith(
-                    t.bos_token.encode()
+                if hasattr(
+                    transformers_tokenizer, "bos_token"
+                ) and reconstructed.startswith(
+                    transformers_tokenizer.bos_token.encode()
                 ):
-                    reconstructed = reconstructed[len(t.bos_token) :]
-            except:
+                    reconstructed = reconstructed[
+                        len(transformers_tokenizer.bos_token) :
+                    ]
```

Comment:
I'd really prefer if we didn't blow up lines because of black...these multi-line things are significantly harder to read imo. Before we continue this wave of refactoring, can we change our default black settings to 1) significantly increase the line length limit to mitigate these types of things and 2) only black code on diffs (which I believe is possible)?

Reply:
Line limit bumped for `black`